docs/tutorials.md

```diff
@@ -5,19 +5,22 @@ layout: "links"
 
 # Tutorials
 
-##  Multi-Agent Deep Reinforcement Learning in 13 Lines of Code Using PettingZoo
-[https://towardsdatascience.com/multi-agent-deep-reinforcement-learning-in-15-lines-of-code-using-pettingzoo-e0b963c0820b](https://towardsdatascience.com/multi-agent-deep-reinforcement-learning-in-15-lines-of-code-using-pettingzoo-e0b963c0820b)
+## Official tutorial
```

Comment:
Could we rename it from Official Tutorial to something like Recommended Start or something similar? The issue is that we're not the maintainers of SB3, and SB3 has had historically very slow updates to support the latest Gym versions and by extension PZ.

By definition, all the tutorials are actually pretty much the same, just done using different RL libraries (which we are not the maintainers for) or no library at all in the case of Tic Tac Toe.